src/BulletGauge/BulletGauge.tsx

```diff
@@ -50,6 +50,7 @@ const BulletGauge = ({ title, value, suffix }: BulletGaugeProps) => {
       ]}
       layout={{
         font: {
+          family: "Montserrat, sans-serif",
```

Comment:
Any reason this component includes the sans-serif family fallback but the other two don't?

Reply:
All looks fine apart from this question.